src/interfaces/element.ts

```diff
@@ -0,0 +1,6 @@
+export interface Element {
+  name?: string
+  widget: string
+  position: number
```

Comment:
- [ ] What is `widget`? It's worth it to add a line comment.
- [ ] What is `position`? It's worth it to add a line comment.